### PR TITLE
DAS-1261 - Add UAT Trajectory Subsetter to services.yml.

### PR DESCRIPTION
**Pull request by owenlittlejohns**
This PR adds the Data Services Segmented Trajectory Subsetter to Harmony. This PR should allow the Trajectory Subsetter to run both via Argo and Harmony Turbo (I've tested it with both locally).

The Trajectory Subsetter can perform:

* Variable subsetting.
* Bounding box subsetting.
* Shape file subsetting.
* Temporal subsetting.
* (In future I think there are plans to enable the reformatting capabilities, too)

Disclaimer for anyone wanting to check this out - the granules in the test collection (C1242267295-EEDTEST) are pretty large (≥ 600 MB, sometime in excess of 1 GB). I found my download speed for my local connection to be _really_ slow.

### CODE REVIEW

**Comment by chris-durbin** [comment]
The changes look good to me. Let me update the harmony-ci-cd repo to set it up to deploy the trajectory subsetter and test it out in sandbox before we merge this in.

File: env-defaults [hunk 1]
```diff
@@ -275,6 +275,14 @@ SDS_MASKFILL_REQUESTS_MEMORY=128Mi
 SDS_MASKFILL_LIMITS_CPU=128m
 SDS_MASKFILL_LIMITS_MEMORY=512Mi
 
+TRAJECTORY_SUBSETTER_IMAGE=sds/trajectory-subsetter:latest
+TRAJECTORY_SUBSETTER_IMAGE_PULL_POLICY='IfNotPresent'
+TRAJECTORY_SUBSETTER_PARALLELISM=2
+TRAJECTORY_SUBSETTER_REQUESTS_CPU=128m
+TRAJECTORY_SUBSETTER_REQUESTS_MEMORY=128Mi
+TRAJECTORY_SUBSETTER_LIMITS_CPU=128m
+TRAJECTORY_SUBSETTER_LIMITS_MEMORY=512Mi
```
Comment (chris-durbin) [inline env-defaults:284]: I'm impressed the subsetter worked for a >600 MB granule with 512MB of memory for the service. Does 512MB seem like enough memory to you?

**Review by bilts — APPROVED** [review]
Approved.  @chris-durbin is there anything we need to do on our end to let this run with Turbo?

File: env-defaults (hunk 1 above)
Comment (owenlittlejohns) [reply env-defaults:284]: Preserving offline conversation: Chris ran a request in the Sandbox environment. Even though the granule was > 600 MB, he saw the pod memory usage max out at 216MB.

**Review by chris-durbin — APPROVED** [review]

**Pull request merged**